libs/execution_context/ebpf_native.c

```diff
@@ -170,16 +186,18 @@ _ebpf_native_clean_up_maps(_In_reads_(map_count) _Frees_ptr_ ebpf_native_map_t*
             ebpf_free(map->pin_path.value);
 #pragma warning(pop)
         }
-        if (map->handle != ebpf_handle_invalid) {
+        if (close_handles && map->handle != ebpf_handle_invalid) {
             ebpf_assert_success(ebpf_handle_close(map->handle));
         }
+        map->handle = ebpf_handle_invalid;
```

Comment:
should this be inside the `if` above?
If not, add a comment explaining why.

Reply:
Fixed.